flake.nix

```diff
@@ -0,0 +1,33 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixOS/nixpkgs";
```

Comment:
can you just add the overlays as `nixpkgs`? 😛

Reply:
We're getting 2 nixpkgs in the flakes.lock then which is annoying :sweat_smile: